### PR TITLE
Added GitHub profile link to my name

### PR DESCRIPTION
**Pull request by accakks**

### CODE REVIEW

**Comment by maddhruv** [comment]
Please add the _stats_ page link too

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Microsoft is participating in Hacktoberfest this year! Details [here](https://op
 | --- | --- | --- | --- |
 | Dhruv Jain | IV | CSE | [@maddhruv](https://hacktoberfest.digitalocean.com/stats/maddhruv) |
 | [Abhilash Tiwari](https://github.com/abhilashk433/) | III | CSE | |
-| Aakanksha Jain | III | CSE | |
+| [Aakanksha Jain](https://github.com/accakks/) | III | CSE | |
```
Comment (maddhruv) [inline README.md]: Please add the stats page link too

Comment (accakks) [reply README.md]: Yes adding.

Comment (accakks) [reply README.md]: Done

**Pull request merged**